lib/pbench/server/api/__init__.py

```diff
@@ -147,39 +147,41 @@ def __init__(self):
 
     def post(self):
         global app
-        graphQL = f"{app.config_graphql.get('host')}:{app.config_graphql.get('port')}"
+        graphQL = (
+            f"http://{app.config_graphql.get('host')}:{app.config_graphql.get('port')}"
+        )
         json_data = request.get_json(silent=True)
 
         if not json_data:
-            app.logger.error("GraphQL: Invalid json object %s", request.url)
+            app.logger.warning("GraphQL: Invalid json object {}", request.url)
             abort(400, message="GraphQL: Invalid json object in request")
 
         try:
             # query GraphQL
-            response = requests.post(graphQL, json=json_data)
-
-            # construct response object
-            response = make_response(response.text)
-            response = _corsify_actual_response(response)
-
+            gql_response = requests.post(graphQL, json=json_data)
         except requests.exceptions.ConnectionError:
             app.logger.exception("Connection refused during the GraphQL post request")
-            abort(500, message="Network problem, could not post to GraphQL Endpoint")
+            abort(503, message="Network problem, could not post to GraphQL Endpoint")
```

Comment:
504 would probably be more correct here as it assigns the root problem to an "upstream" server rather than to us.

Reply:
Yes, `503` is not right, but `504` is `Gateway Timeout`, perhaps `502`, `Bad Gateway`?  https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html